health-services/libraries/health-services-models/src/main/java/org/egov/common/models/project/TaskResource.java

```diff
@@ -48,7 +48,7 @@ public class TaskResource   {
 
     @JsonProperty("quantity")
     @NotNull
-    private Long quantity = null;
+    private Double quantity = null;
```

Comment:
Does this take negative value as well? as per the code it will, are we also checking for negative on the regex validation end?

Reply:
Both the Regex patterns check for negative value